tfexec/internal/e2etest/apply_test.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/hashicorp/terraform-exec/tfexec"
 )
 
+var (
+	validateMinApplyDestroyVersion = version.Must(version.NewVersion("0.15.2"))
```

Comment:
```suggestion
	applyDestroyMinVersion = version.Must(version.NewVersion("0.15.2"))
```

Reply:
would be great to merge this, i can fork the author's branch if that's helpful